docs/conf.py

```diff
@@ -36,23 +34,19 @@
 author = f"{project} project contributors"
 copyright = author  # pylint:disable=redefined-builtin
 
-# fmt: off
-# The short X.Y version
-version = ".".join(
-    get_scm_version(
-        local_scheme="no-local-version",
-    ).split(".")[:3],
-)
+NAVIGATOR_VERSION = get_version("ansible_navigator")
+
+# The short X.Y.Z version
```

Comment:
but it's totally wrong? :)

Reply:
It's sort of a balance. I'd maybe put an additional inline comment to explain the rest?

Reply:
I'm assuming you mean this:  https://github.com/sphinx-doc/sphinx/blob/a8eb1aab72a514b7257c780c0c57432732a702fb/sphinx/templates/quickstart/conf.py_t

I'm going to leave this, for exactly the reason you mentioned, when diffed, it will highlight the the deviation from the tempalte.

Reply:
I'll compromise, brb